readme.md

```diff
@@ -48,12 +48,13 @@ theBody.style.backgroundColor = "red";
 Why does the following code snippet throw an error, and what would fix the error?
 
 ```css
-body{
+body {
   background-color = "red";
 }
 ```
 
-> Your answer...
+> The color should be without quotes,
+background-color = red;
```

Comment:
Oh, I didn't see this before I commented on your change above. I see you got 1/2 of it here, but the colon is still important.